STEP_Project/tests/popupTest.js

```diff
@@ -37,21 +42,20 @@ describe('Testing enabling and disabling of buttons depending on pageType', ()=>
   beforeEach(function() 
   {
     //back to default for future tests
+    const arButton = document.createElement('BUTTON')
+    arButton.setAttribute("id", "addRemove");
+    const sButton = document.createElement('BUTTON')
+    sButton.setAttribute("id", "select");
+    document.body.appendChild(arButton);
+    document.body.appendChild(sButton);
     s = document.getElementById("select");
     ar = document.getElementById("addRemove");
   });
 
   afterEach(function() 
   {
-    //back to default for future tests
-    if (s !== null)
-    {
-      s.disabled = false;
-    }
-    if (ar !== null)
-    {
-      ar.disabled = false;
-    }
+    document.getElementById("select").remove();
+    document.getElementById("addRemove").remove();
```

Comment:
Maybe move this to a separate function and call that function here.

Reply:
moved to _restoreDOM_ function